patterns/ecr-pull-through-cache/addons.tf

```diff
@@ -1,3 +1,9 @@
+locals {
+  ecr_account_id = var.ecr_account_id != "" ? var.ecr_account_id : data.aws_caller_identity.current.account_id
```

Comment:
lets remove account ID and region entirely - we can get this data from the provider/data sources

Reply:
I kept those as different from the main region/account to illustrate that's possible to use an ECR on a different region/account, not required to be the same as the cluster deployment.